augur/analyze.py

```diff
@@ -138,6 +158,145 @@ def __init__(self, config, likelihood=None, tools=None, req_params=None,
+            for var in self.gprior_pars:
+                _val = self.config['gaussian_priors'][var]
+                self.gpriors.append(_val)
+
+    def get_Om(self):
+        """
+        Function that calculates the fiducial value of Omega_m from the input fiducial cosmology.
+
+        Returns:
+        --------
+        Om : float
+             Fiducial value of Om evaluated about pars_fid.
+        """
+        if self.Om is None:
+            Om = 0.0
+            if 'Omega_c' not in self.var_pars:
+                raise ValueError('Require Omega_c to be specified \
+                                 when transforming the Fisher matrix to Omega_m')
+            if 'Omega_c' in self.pars_fid.keys():
+                Om += self.pars_fid['Omega_c']
+            if 'Omega_b' in self.pars_fid.keys():
+                Om += self.pars_fid['Omega_b']
+            if 'm_nu' in self.pars_fid.keys():
+                m_nu = self.pars_fid['m_nu']
+                if m_nu > 0.0:
+                    if 'h' not in self.pars_fid.keys():
+                        raise ValueError('Require h to be specified \
+                                         when transforming the Fisher matrix to Omega_m with m_nu')
+                    h = self.pars_fid['h']
+                    Om += m_nu/h/h/93.14
+            self.Om = Om
+        return self.Om
+
+    def get_S8(self):
+        """
+        Function that calculates the fiducial value of S8 from the input fiducial cosmology.
+
+        Returns:
+        --------
+        S8 : float
+             Fiducial value of S8 evaluated about pars_fid.
+        """
+        if self.S8 is None:
+            S8 = 0.0
+            Om = self.get_Om()
+            if 'sigma8' in self.pars_fid.keys():
+                sigma_8 = self.pars_fid['sigma8']
+                S8 = np.sqrt(Om/0.3) * sigma_8
+            else:
+                raise ValueError('Require sigma8 to be specified \
+                                 when transforming the Fisher matrix to S8')
+            self.S8 = S8
+        return self.S8
+
+    def Jacobian_transform(self):
+        """
+        Function that returns the Jacobian to transform the basis from:
+        - Omega_c -> Omega_m
+        - sigma8 -> S8
+        and will replace its value in the Fisher matrix.
+        If neither transform is specified, this function will return the Identity matrix
+        to keep all subsequent operations consistent.
+
+        Returns:
+        --------
+        J : np.ndarray
+            Jacobian transformation matrix evaluated about pars_fid.
+        """
+        if self.J is None:
+            J = np.identity(len(self.x))
+            if self.transform_Omega_m:
+                Om = self.get_Om()
+                ind_c = None
+                if 'Omega_c' in self.var_pars:
+                    ind_c = np.where(np.array(self.var_pars) == 'Omega_c')[0][0]
+                if ind_c is None:
```

Comment:
Technically speaking I think that line 235 would have failed already if `Omega_c` is None